### PR TITLE
feat(votes): mocked out routes for voting functionality

### PR DESCRIPTION
**Pull request by bryanrichie**

### CODE REVIEW

File: src/client/components/CommentCount.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import _ from 'lodash';
+import { Text } from '@chakra-ui/react';
+
+interface Props {
+  commentCount: string;
```
Comment (sbousamra) [inline src/client/components/CommentCount.tsx:6]: how come this guy isn't a number out of curiosity?

File: src/client/components/CommentCount.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+export const CommentCount = (props: Props) => {
+  if (_.toInteger(props.commentCount) < 1) {
+    return <Text color={'gray.500'}>Comment</Text>;
+  } else if (_.toInteger(props.commentCount) == 1) {
```
Comment (sbousamra) [inline src/client/components/CommentCount.tsx:12]: == isn't the same as ===. You want to use === for equality checks

Comment (sbousamra) [reply src/client/components/CommentCount.tsx:12]: Also am i missing something? You're returning the same thing in every case? What's the point of the if conditions?

File: src/client/components/Post.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+interface Props {
+  postId: string;
+  post: DatabasePost;
+}
+
+export const Post = (props: Props) => {
+  const { postId, post } = props;
+
+  const postBg = useColorModeValue('white', 'gray.700');
+  const postBorder = useColorModeValue('gray.700', 'white');
```
Comment (sbousamra) [inline src/client/components/Post.tsx:18]: these are back to front? Keep consistent unless im missing something?

File: src/server/services/post.service.ts [hunk 4]
```diff
@@ -18,20 +22,20 @@ export class PostService {
     return this.databaseService.createPost(post);
   }
 
-  deletePost(postId: string, userId: string) {
-    return this.databaseService.deletePost(postId, userId);
-  }
+  // deletePost(postId: string, userId: string) {
```
Comment (willemclarke) [inline src/server/services/post.service.ts:25]: leftover comment

**Pull request merged**